apps/emqx_statsd/src/emqx_statsd_schema.erl

```diff
@@ -53,11 +54,13 @@ fields("statsd") ->
 desc("statsd") -> ?DESC(statsd);
 desc(_) -> undefined.
 
-server(type) -> emqx_schema:host_port();
-server(required) -> true;
-server(default) -> "127.0.0.1:8125";
-server(desc) -> ?DESC(?FUNCTION_NAME);
-server(_) -> undefined.
+server() ->
+    Meta = #{
+        default => "127.0.0.1:8125",
+        required => false,
```

Comment:
This was required before; with this, won't the validator accept an empty host:port?

Reply:
deleted `default`. 
required fields should not have default values.
fields with default value should not be required.

Reply:
should probably add an assertion in hocon.git

Reply:
turns out it was in fact NOT required before my change. because the default value is filled-in before the null-check.